### ISSUE TITLE
Copy in the description window does not update main list

### ISSUE THREAD

**Issue by Rainroom69** [sabrogden/Ditto#69]
... at once.  the new clip only appears when I close the main list and make it visible again.  version 3.24.182 portable.
I remember with some older versions a copy of selected text in the description window (F3) appeared immediately in main list.

**Comment by sabrogden** [comment]
Yes this was changed because before if you copied something in the description window the description window would close and the list would refresh and you would loose what you were looking at.  Now everything stays where it is and the copied clip will show up in the list on next refresh (next copy or search or press F5 to manually refresh)

**Comment by Chealer** [comment]
I cannot reproduce this with 03.25.113. Can you? If so, please provide reproduction steps.
